### ISSUE TITLE
Gene page Location widget JBrowse inset: can we get some more visual padding/space?

### ISSUE THREAD

**Issue by chris-grove** [WormBase/website#5900]
@scottcain 
This is kind of a general issue I've been having with JBrowse: There seems to so much effort in JBrowse to be economical with visual space that often I find it difficult to see what I want to see. For example, track labels/names are often getting the in the way of features that I want to see. I know that there's a button to hide the track names, which is nice, but (1) users need to know that button exists and where to find it and (2) it seems like this wouldn't be an issue if the track label had its own dedicated horizontal space, like it does in GBrowse. My guess is that this was a deliberate design decision made by the JBrowse developers to save space, but I thought I'd express my thoughts on it.

My specific issue for this ticket is the JBrowse inset image in the Location widget on gene pages.

For example, on the WormBase pop-1 gene page:

http://www.wormbase.org/species/c_elegans/gene/WBGene00004077#09--10

in the Location widget we have the JBrowse inset image of the pop-1 locus and the GBrowse inset image:

![screen shot 2017-10-12 at 9 53 15 am](https://user-images.githubusercontent.com/611382/31500169-bb8a5e70-af34-11e7-88f0-9791ddd5db49.png)

As you can see from the screenshot, the 5' portion of the pop-1 gene model is obstructed by the "Full-screen view", "Full Page" etc. buttons because the gene model sits at the very top of the track and there's not enough margin/space to allow the entire model to be visible. In this context, I still prefer the GBrowse image inset, even though it is static, because I can clearly see all of the gene's features.

Another example is the hif-1 gene page:

http://www.wormbase.org/species/c_elegans/gene/WBGene00001851#09--10

![screen shot 2017-10-12 at 10 10 11 am](https://user-images.githubusercontent.com/611382/31500508-96ec29d0-af35-11e7-9516-ea5394e520dc.png)

and the daf-16 page:

http://www.wormbase.org/species/c_elegans/gene/WBGene00000912#09--10

![screen shot 2017-10-12 at 10 11 27 am](https://user-images.githubusercontent.com/611382/31500583-c5d98f44-af35-11e7-9f47-ce70caab666f.png)

So, can we get some more padding/margin/space around the gene model so that the JBrowse image of the locus is always visible in its entirety?

**Comment by scottcain** [comment]
Hi @chris-grove ,

I agree.  When I accepted the "hide track label" button a few years ago, I hoped it would be a temporary solution to better track label placement.  At the moment, there isn't a way to reposition the track labels, but I am lobbying for it.

**Comment by chris-grove** [comment]
@scottcain 
OK, so is there nothing we can do now about the JBrowse inset on gene page Location widgets? This will require a major change of JBrowse first? Should I just leave this ticket open as a feature request?

**Comment by scottcain** [comment]
@chris-grove Yes, leave it open as a feature request, as the only fix is for JBrowse to change in a fairly major way (at least, someone would have to spend a fair amount of time writing a plugin that I don't currently know how to write (which is why I estimate that it would take a long time)).  Of course the other option is to pitch JBrowse in the location widget and replace it with the AGR widget :-)

**Comment by chris-grove** [comment]
@scottcain 
OK, I'll leave it in as a feature request as long as we are displaying the JBrowse inset on the gene page. If we do get rid of it, then we can close this issue. Thanks!
